lib/foreman_maintain/cli/system_report.rb

```diff
@@ -0,0 +1,62 @@
+module ForemanMaintain
+  module Cli
+    class SystemReport < Base
+      class << self
+        def limit_table_size_option
+          option ['-s', '--size'], '[>=1mb|<=1MB]',
+                 'Show tables with size specified operator(>= or <= or > or <).'\
+                 'Defaults to >=1MB.' do |size|
+            size = Report::Params::Size.new(size)
+            size.validate!
```

Comment:
@swapab 
I have thought that instead of calling `validate!` method separately here why not to put it in `initialize` method only.

For example - 

```
def initialize(size)
     @size = disintegrate_size(size)
     validate!
end
```

Reply:
That's a good point. But I feel `Object#initialize` method should only care about initializing objects.  Public methods should be called as needed.

Also, in this case, it provides a good readability.